tests/MCryptCompatTest.php

```diff
@@ -1030,4 +1052,30 @@ public function setExpectedException($name, $message = null, $code = null)
             $this->expectExceptionCode($code);
         }
     }
+
+    public function errorHandler($errNo, $errStr, $errFile, $errLine, $errContext)
+    {
+        if ($errStr === $this->expectedErrorMessage && $errNo === $this->expectedErrorLevel)
+        {
+            return;
+        }
+
+        call_user_func($this->previousErrorHandler, $errNo, $errStr, $errFile, $errLine, $errContext);
+    }
```

Comment:
Seems like we'd be better off re-using the existant `setExpectedException` method:

https://github.com/phpseclib/mcrypt_compat/blob/master/tests/MCryptCompatTest.php#L1014